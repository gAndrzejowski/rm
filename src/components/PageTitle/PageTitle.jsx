import React from 'react';
import styles from './PageTitle.scss';

const PageTitle = () => <p className={styles.title}>netflixroulette</p>;
export default PageTitle;