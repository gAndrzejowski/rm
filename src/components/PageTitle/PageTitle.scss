.title {
  font-weight:bold;
  color: firebrick;
  font-size:1.2rem;
}