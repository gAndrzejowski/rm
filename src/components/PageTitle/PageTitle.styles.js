const styles = {
  title: {
    fontWeight: 'bold',
    color: 'firebrick',
    fontSize: '1.2rem',
  },
};
export default styles;
