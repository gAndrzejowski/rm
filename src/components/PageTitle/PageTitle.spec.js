import React from 'react';
import renderer from 'react-test-renderer';
import PageTitle from './PageTitle';

describe('PageTitle component', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<PageTitle/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
});