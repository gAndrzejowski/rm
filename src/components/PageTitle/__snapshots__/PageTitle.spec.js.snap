// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle component renders correctly 1`] = `
<p
  className="title"
>
  netflixroulette
</p>
`;
